src/modules/IsnsServer.rb

```diff
@@ -363,7 +364,8 @@ def SetStartService(status)
     private
 
     def isnsadm(params, ret_result = false)
-      command = "isnsadm --local #{params}"
+      # cannot shellescape here as more params can be passed
+      command = "/usr/sbin/isnsadm --local #{params}"
```

Comment:
What is the source of these params? Are they correctly escaped?

Reply:
yes, it is private method and used only here. That is why I escape all those isnsadm calls in this file.